ops/ansible/roles/bfd-pipeline/tasks/main.yml

```diff
@@ -62,14 +62,6 @@
   become: true
   tags: 
     - pre-ami
-
-- name: Restart App Server Service
-  service:
-    name: "{{ data_server_appserver_service }}"
-    state: restarted
-  become: true
-  tags:
-    - post-ami
     
```

Comment:
```suggestion

# This task will run for HealthAPT since no tags are specified. 
- name: Restart App Server Service	
  service:	
    name: "{{ data_server_appserver_service }}"	
    state: restarted	
  become: true	
  
```

This is needed for the HealthAPT environment deployment. So I've removed the tag which will fire for HealthAPT who dont specify tags (Ansible defaults to all in this case) and will be skipped for CCS since we use post-ami and pre-ami.